app/views/registration_wizard/dont_have_teacher_reference_number.html.erb

```diff
@@ -13,75 +13,91 @@
+      To send a secure email to the department, please go to the Galaxkey site:
+      <%= govuk_link_to "https://manager.galaxkey.com/services/registerme", "https://manager.galaxkey.com/services/registerme",  target: :_blank, rel: "noopener noreferrer" %>
+      (opens in a new tab)
     </p>
 
     <p class="govuk-body">
       Enter the email address that you wish to sign up with and complete the registration process.
     </p>
 
     <p class="govuk-body">
-      You will be sent an email to activate your account once you have completed this step. You can then send secure emails to the Teaching Regulation Agency using Galaxkey.
+      You'll be sent an email to activate your account once you have completed this step. You can then send secure emails to the Teaching Regulation Agency using Galaxkey.
     </p>
 
-    <h2 class="govuk-heading-m">Secure emails - a quick start guide</h2>
+    <h2 class="govuk-heading-m">Send your ID documents securely</h2>
 
     <p class="govuk-body">
-      The steps you need to take are outlined in the following quick start guide below:
+      Follow these steps to send your documents through Galaxkey:
     </p>
 
-    <ul class="govuk-list govuk-list--bullet">
-      <li>select ‘Compose’ to start a secure email</li>
-      <li>enter <strong>qts.enquiries@education.gov.uk</strong> as the recipient</li>
-      <li>input the ‘Subject’ of the email as: Undertaking an NPQ, do not have a TRN and require a number to be issued</li>
+    <ol class="govuk-list govuk-list--number">
+      <li>Select ‘Compose’ to create a new secure email.</li>
+      <li>Enter ’qts.enquiries@education.gov.uk’ as the recipient.</li>
+      <li>Enter ‘Undertaking an NPQ, do not have a TRN and require a number to be issued’ as the subject line.</li>
       <li>
-        include the following information in your email:
+        Include the following information in your email:
 
         <ul class="govuk-list govuk-list--bullet">
-          <li>state you are undertaking an NPQ, do not have a TRN and require a number to be issued</li>
-          <li>your full legal name (we will use this to create a record for you)</li>
+          <li>your full legal name</li>
           <li>your date of birth</li>
-          <li>an email address for us to respond to you at</li>
-          <li>
-            attach a copy of one of the following types of identification as a file to accompany your submission. Note the restrictions on file size in the provided guidance:
-            <ul class="govuk-list govuk-list--bullet">
-              <li>passport</li>
-              <li>driving license (full or provisional)</li>
-              <li>certificate of residence</li>
-              <li>birth certificate</li>
-            </ul>
-          </li>
+          <li>an email address for the team to reply to</li>
         </ul>
+
+        <li>Attach a scanned copy or photo of one of the following types of ID:
```

Comment:
i don't believe a nested `li` is valid

Reply:
good spot, this wasn't intended, i missed off closing the previous li